app/components/ui/Navigation.module.scss

```diff
@@ -137,6 +139,10 @@
   display: inline-block;
   height: 24px;
   width: 24px;
+
+  @include r_min($break-tablet-s) {
+    display: none !important;
+  }
```

Comment:
The above changes are just to adjust the responsive breakpoints, since this added some width to the nav content.